src/main/java/org/mifos/connector/channel/GSMA_API/GSMADepositAPIController.java

```diff
@@ -0,0 +1,34 @@
+package org.mifos.connector.channel.GSMA_API;
+
+import com.fasterxml.jackson.core.*;
+import com.fasterxml.jackson.databind.*;
+import org.apache.camel.*;
+import org.mifos.connector.channel.utils.Headers;
+import org.mifos.connector.channel.utils.SpringWrapperUtil;
+import org.mifos.connector.common.gsma.dto.*;
+import org.springframework.beans.factory.annotation.*;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class GSMADepositAPIController implements GSMADepositAPI {
+
+    @Autowired
+    private ProducerTemplate producerTemplate;
+
+    @Autowired
+    ObjectMapper objectMapper;
+
+    @Override
+    public RequestStateDTO gsmadeposit(String tenant, String correlationId, GSMATransaction requestBody) throws JsonProcessingException {
+        org.mifos.connector.channel.utils.Headers headers = new Headers.HeaderBuilder()
+                .addHeader("Platform-TenantId", tenant)
```

Comment:
why are we mentioning static string here? Shouldn't it be referenced form camelProperties??